packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-role-list/aria-role-list.docs.mdx

```diff
@@ -0,0 +1,137 @@
+import { Meta, Canvas, Source } from '@storybook/addon-docs/blocks';
+import * as CCR from './aria-role-list.stories';
+
+import DefaultExample from './examples/defaultExample.sample.html?raw';
+import DefaultExample2 from './examples/defaultExample2.sample.html?raw';
+import DefaultExample3 from './examples/defaultExample3.sample.html?raw';
+import WorkingExampleIIa from './examples/workingExample-IIa.sample.html?raw';
+import WorkingExampleIIb from './examples/workingExample-IIb.sample.html?raw';
+import WorkingExampleIIIa from './examples/workingExample-IIIa.sample.html?raw';
+import WorkingExampleIIIb from './examples/workingExample-IIIb.sample.html?raw';
+import WorkingExampleIVa from './examples/workingExample-IVa.sample.html?raw';
+import WorkingExampleIVb from './examples/workingExample-IVb.sample.html?raw';
+import WorkingExampleIVc from './examples/workingExample-IVc.sample.html?raw';
+
+<Meta of={CCR} />
+
+# role="list"
+
+<div className="lead">
+  The `role="list"` tells assistive technologies that an element represents a list, which is
+  expected to contain related items identified with `role="listitem"`.
+</div>
+
+### I. Referencing Within the Same DOM Tree
+
+• <b>Both `list` parent and `listitem` children in Light DOM</b> ✔️
+
+<Source language="html" dark code={DefaultExample}></Source>
+
+<Canvas of={CCR.ExampleHTML} sourceState="hide" />
+
+<p>
+  This represents the default structure, where a `role="list"` element contains direct children with
+  `role="listitem"`.
+</p>
+
+• <b>Both `list` parent and `listitem` children in the same Shadow DOM</b> ✔️
+
+<Source language="html" dark code={DefaultExample2}></Source>
+
+<Canvas of={CCR.Example1a} sourceState="hide" />
+
+<p>
+  Same as above, but contained within a custom component’s Shadow DOM. The `role="list"` still has
+  direct `role="listitem"` children, so the structure remains valid.
+</p>
+
+• <b>`list` role on the host and slotted `listitem` children</b> ✔️
+
+<Source language="html" dark code={DefaultExample3}></Source>
+
+<Canvas of={CCR.Example1b} sourceState="hide" />
+
+<p>
+  Similarly, in the case of a component with `role="list"` set on the host and slotted `listitems`, the
+  relationship is preserved correctly.
+</p>
+
+### II. Parent in the Light DOM-children Into The Shadow DOM
+
+• <b>Light DOM → Shadow DOM</b> ✔️
+
+<Source language="html" dark code={WorkingExampleIIa}></Source>
+
+<Canvas of={CCR.Example2a} sourceState="hide" />
+
+<p>
+  In the case a component with `role="list"` in the host with `listitem` elements within its Shadow
+  DOM, the relationship is valid.
+</p>
```

Comment:
But why? Should be explained, as it is seems to be an exception.
Or what's the difference to the aria attributes, which can't cross different DOM borders.